app/controllers/projects_controller.rb

```diff
@@ -1,5 +1,14 @@
 class ProjectsController < ApplicationController
   def index
     @presenter = ProjectsPresenter.new(params)
+    @props = {
+      tags: tags
```

Comment:
I'm surprised to not see a `projects` key here. 🤔 Is that `@presenter` ivar used somewhere?

Reply:
Will send a few more PRs over the coming week to add project views!

Reply:
Woot!